MIP/loader/loader.m

```diff
@@ -20,70 +20,99 @@
     return user_data_path;
 }
 
+BOOL should_inject_bundle(
+    NSBundle *tweakBundle,
+    NSString *mainExecutableName,
+    NSArray<NSString *> *globalyDisabledBundles
+) {
+    NSDictionary *plist = tweakBundle.infoDictionary;
+    BOOL blacklistMode = [plist[@"MIPUseBlacklistMode"] boolValue];
+    
+    // Skip tweak if it is globally disabled
+    if ([globalyDisabledBundles containsObject:tweakBundle.bundleIdentifier]) {
+        // Not affected by blacklist mode
+        return NO;
+    }
+    
+    // Skip tweak if a matching bundle is excluded
+    for (NSString *entry in plist[@"MIPExcludedBundleNames"]) {
+        if (CFBundleGetBundleWithIdentifier((CFStringRef)entry)) {
+            // Match found; invert if blacklist mode enabled
+            return NO;
+        }
+    }
+    
+    // Check if process matches bundle filter
+    for (NSString *entry in plist[@"MIPBundleNames"]) {
+        if (CFBundleGetBundleWithIdentifier((CFStringRef)entry)) {
+            // Match found; invert if blacklist mode enabled
+            return blacklistMode ? NO : YES;
+        }
+    }
```

Comment:
Calling `CFBundleGetBundleWithIdentifier` with the given bundle ID will return true if the bundle was linked against